### PR TITLE
Cyprit-opencl for brew

### PR DESCRIPTION
**Pull request by gerardbriscoe**

### CODE REVIEW

File: Library/Formula/cpyrit-opencl.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CpyritOpencl < Formula
+  desc "OpenCL support for Pyrit."
+  homepage "https://code.google.com/p/pyrit/wiki/Installation"
+  url "https://pyrit.googlecode.com/files/cpyrit-opencl-0.4.0.tar.gz"
+  sha256 "aac593bce3f00ea7fd3d558083dbd7e168332a92736e51a621a0459d1bc042fa"
+
+  depends_on :python
```
Comment (DomT4) [inline Library/Formula/cpyrit-opencl.rb]: Should be `depends_on :python if MacOS.version <= :snow_leopard`

File: Library/Formula/cpyrit-opencl.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  depends_on 'pyrit'
```
Comment (DomT4) [inline Library/Formula/cpyrit-opencl.rb]: Double quotes everywhere.

File: Library/Formula/cpyrit-opencl.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  def install
+    system "python", "setup.py", "build"
+    system "python", "setup.py", "install", "--prefix=#{prefix}"
```
Comment (DomT4) [inline Library/Formula/cpyrit-opencl.rb]: Should be set up something like [this](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/jrnl.rb#L62-L66) if possible.

File: Library/Formula/cpyrit-opencl.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
```
Comment (DomT4) [inline Library/Formula/cpyrit-opencl.rb]: No need for an empty line here.

File: Library/Formula/cpyrit-opencl.rb (hunk 1 above)
Comment (tdsmith) [inline Library/Formula/cpyrit-opencl.rb]: No, this is a library, not an application.

**Comment by tdsmith** [comment]
This should be submitted to Homebrew/homebrew-python.

File: Library/Formula/cpyrit-opencl.rb (hunk 1 above)
Comment (tdsmith) [inline Library/Formula/cpyrit-opencl.rb]: No, you're right, I'm wrong.

**Comment by tdsmith** [comment]
All of the pyrit formulas could be migrated into Homebrew/homebrew from Homebrew/homebrew-python if you prefer but they should all be in the same place. Does it make sense to ship this separately from the pyrit formula or should we always bundle it?

File: Library/Formula/cpyrit-opencl.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/cpyrit-opencl.rb]: Ah, Sorry. I didn't know we handled python library formula differently in terms of the dep:question:

**Comment by gerardbriscoe** [comment]
I am new to brew. I followed the example provided and cpyrit-cuda as much as I could.

**Comment by DomT4** [comment]
> This should be submitted to Homebrew/homebrew-python.

Let's do this, Thanks!

**Pull request closed without merging**